components/Chips/src/MDCChipView.m

```diff
@@ -400,6 +417,7 @@ - (void)setInkColor:(UIColor *)inkColor forState:(UIControlState)state {
 - (void)updateInkColor {
   UIColor *inkColor = [self inkColorForState:self.state];
   self.inkView.inkColor = inkColor ? inkColor : self.inkView.defaultInkColor;
+  self.rippleView.rippleColor = inkColor ?: [UIColor colorWithWhite:1 alpha:(CGFloat)0.12];
```

Comment:
Extract the 0.12 value to a static constant in this file please.

Reply:
Done.